spec/indexers/hyrax/valkyrie_work_indexer_spec.rb

```diff
@@ -3,22 +3,48 @@
 require 'hyrax/specs/shared_specs'
 
 RSpec.describe Hyrax::ValkyrieWorkIndexer do
-  let(:indexer) { described_class.new(resource: resource) }
   let(:resource) { FactoryBot.valkyrie_create(:hyrax_work) }
   let(:indexer_class) { described_class }
 
-  it_behaves_like 'a Hyrax::Resource indexer'
-  it_behaves_like 'a Core metadata indexer'
-  it_behaves_like 'a permission indexer'
-  it_behaves_like 'a visibility indexer'
+  it_behaves_like 'a Work indexer'
```

Comment:
All shared specs representing the minimal required parts of a work are now defined and tested in the shared spec 'a Work indexer'.